srcs/utils/system_exception.cpp

```diff
@@ -0,0 +1,12 @@
+#include "system_exception.hpp"
+
+namespace utils {
+
+SystemException::SystemException(const std::string &error_message, int error_number)
+	: runtime_error(error_message), error_number_(error_number) {}
```

Comment:
細かいのですが std 忘れ (何でエラーにならないんだろう…？)
```suggestion
SystemException::SystemException(const std::string &error_message, int error_number)
	: std::runtime_error(error_message), error_number_(error_number) {}
```